src/common/intl/locales/fr.js

```diff
@@ -1644,17 +1643,17 @@ export default {
   // Original text: "Host up-to-date!"
   hostUpToDate: 'Hôte à jour !',
 
-  // Original text: 'Non-recommended patch install'
-  installPatchWarningTitle: undefined,
+  // Original text: "Non-recommended patch install"
+  installPatchWarningTitle: 'Installation de patch non recommandée',
 
-  // Original text: 'This will install a patch only on this host. This is NOT the recommended way: please go into the Pool patch view and follow instructions there. If you are sure about this, you can continue anyway'
-  installPatchWarningContent: undefined,
+  // Original text: "This will install a patch only on this host. This is NOT the recommended way: please go into the Pool patch view and follow instructions there. If you are sure about this, you can continue anyway"
+  installPatchWarningContent: "Ceci installera un patch sur cet hôte uniquement. Ce n'est pas la procédure recommandée : veuillez vous rendre sur la page des patches du pool et suivre les instructions. Si vous êtes certain de ce que vous faites, vous pouvez tout de même continuer.",
```

Comment:
I think we can simplify here: *Installer un patch sur un hôte seul est déconseillé. Il est recommandé d'aller sur la page du pool et de faire l'installation sur tout les hôtes.*